src/Arcus.WebApi.Unit/Security/Authorization/JwtTokenReaderTests.cs

```diff
@@ -28,5 +29,53 @@ public void Constructor_WithApplicationId_Succeeds()
             // Assert
             Assert.NotNull(jwtTokenReader);
         }
+
```

Comment:
Is there a way to test the validation token more thoroughly?

Reply:
Like, maybe generate a valid token (--> pass), and generate a token and alter some values (--> fail).

Reply:
That's mentioned on issue and why it's not closed yet, more to come